internal/processing/helpers.go

```diff
@@ -54,6 +55,18 @@ func setAccessRuleKey(hasPathDuplicates bool, rule rulev1alpha1.Rule) string {
 	return rule.Spec.Match.URL
 }
 
+func setAuthorizationPolicyKey(hasPathDuplicates bool, ap *istiosecurityv1beta1.AuthorizationPolicy) string {
```

Comment:
shouldn't this method be called `getAuthorizationPolicyKey` , `set` is a bit confusing, same as the one above (which is old)?